projects/js-packages/components/components/jetpack-footer/index.tsx

```diff
@@ -19,17 +22,72 @@ const JetpackIcon: React.FC = () => (
  * @returns {React.ReactNode} JetpackFooter component.
  */
 const JetpackFooter: React.FC< JetpackFooterProps > = ( {
-	a8cLogoHref = 'https://automattic.com',
 	moduleName = __( 'Jetpack', 'jetpack' ),
 	className,
 	moduleNameHref = 'https://jetpack.com',
 	menu,
+	siteAdminUrl,
+	onAboutClick,
+	onPrivacyClick,
+	onTermsClick,
 	...otherProps
 } ) => {
 	const [ isSm ] = useBreakpointMatch( 'sm', '<=' );
 	const [ isMd ] = useBreakpointMatch( 'md', '<=' );
 	const [ isLg ] = useBreakpointMatch( 'lg', '>' );
 
+	const { isActive, connectedPlugins } = useSelect(
+		select => {
+			const connectionStatus = select( CONNECTION_STORE_ID ) as {
+				getConnectedPlugins: () => { slug: string }[];
+				getConnectionStatus: () => { isActive: boolean };
+			};
+
+			return {
+				connectedPlugins: connectionStatus?.getConnectedPlugins(),
+				...connectionStatus.getConnectionStatus(),
+			};
+		},
+		[ CONNECTION_STORE_ID ]
+	);
+
+	const areAdminLinksEnabled =
+		siteAdminUrl &&
```

Comment:
Ideally, `siteAdminUrl` would be retrieved from an initial state, as is `connectedPlugins`. I haven't found a way of retrieving basic site information that works across packages yet.

Reply:
We now have `getAdminUrl` and other such utils available in `@automattic/jetpack-script-data`

```ts
import { getAdminUrl, getJetpackAdminPageUrl } from '@automattic/jetpack-script-data';

const aboutPageUrl = getAdminUrl( 'admin.php?page=jetpack_about' );

const privacyPageUrl = getJetpackAdminPageUrl( '#/privacy' );

```

Reply:
Likewise, you can use the same package to get the list of connected plugins and the status.

```ts
import { getScriptData } from '@automattic/jetpack-script-data';

const { connectedPlugins, connectionStatus } = getScriptData().connection;

```